System.IO.Abstractions.TestingHelpers.Tests/MockFileSystemTests.cs

```diff
@@ -54,6 +70,19 @@ public void MockFileSystem_GetFile_ShouldReturnFileRegisteredInConstructorWhenPa
             Assert.AreEqual(file1, result);
         }
 
+        [Test]
+        public void MockFileSystem_AddFile_ShouldHandleNullFileDataAsEmpty()
+        {
+            var fileSystem = new MockFileSystem(new Dictionary<string, MockFileData>
+            {
+                { @"c:\something\nullish.txt", null }
+            });
+
+            var result = fileSystem.GetFile(@"c:\SomeThing\nullish.txt").TextContents;
```

Comment:
I’d suggest using `IFileSystem` APIs for the assert parts.

Reply:
What do you mean?

Reply:
I mean using some method exposed via `IFileSystem` to get the file contents, in this case probably `fileSystem.File.ReadAllText`.